guardian/grid-feeds
# Initial Application - Associated Press API feed

## What does this change?
This creates a basic Play application that calls the Associated Press (AP) API, downloads images and writes them to an S3 bucket specified in config. Note, this is not production-ready, but gives us a minimal service which will allow us to test in CODE.

## How to test
Run `sbt run` in the root of the project. The service should spin up, check this by hitting `http://localhost:8855/management/healthcheck` in the browser and `http://localhost:8855/start` to start the service. You will need a config file `/etc/gu/grid-feeds.private.conf` and media-service AWS credentials.
